Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "sanctum-solana-serde"
+version = "0.1.0"
```

lets just go with v1
```suggestion
version = "1.0.0"
```